library/src/main/java/in/krharsh17/barview/Constants.java

```diff
@@ -0,0 +1,9 @@
+package in.krharsh17.barview;
+
+public interface Constants {
+
+    String label_text_color = "#424242";
+    String value_text_color = "FFFFFF";
+    String ripple_color = "EEEEEE";
+    String charArray = "0123456789ABCDEF";
```

Comment:
Please use `camelCasing` in naming all variables throughout the project. For more details, refer to [this](https://en.wikipedia.org/wiki/Camel_case).